bundles/org.eclipse.equinox.p2.tests/src/org/eclipse/equinox/p2/tests/artifact/repository/CompositeArtifactRepositoryTest.java

```diff
@@ -611,8 +612,10 @@ public void perform(IArtifactRepository sourceRepository, IProgressMonitor monit
 		assertThat(status, is(statusWithMessageWhich(containsString("while reading artifacts from child repositories"))));
 
 		// bug 391400: status should point to repository with problem
-		String brokenChildURI = repositoryURI.toString() + "child";
-		assertThat(Arrays.asList(status.getChildren()), hasItem(statusWithMessageWhich(containsString(brokenChildURI))));
+		assertThat(Arrays.asList(status.getChildren()),
+				hasItem(statusWithMessageWhich(containsString("java.io.FileNotFoundException"))));
```

Comment:
This is the actual message:

> An error occurred copying file `<location>`\p2\bundles\org.eclipse.equinox.p2.tests\testData\artifactRepo\composite\errorInChild\child\plugins\plugin_1.0.0.jar.

The associated exceptions messages is

> java.io.FileNotFoundException: `<location>`\\git\p2\bundles\org.eclipse.equinox.p2.tests\testData\artifactRepo\composite\errorInChild\child\plugins\plugin_1.0.0.jar (The system cannot find the path specified)

```suggestion
				hasItem(statusWithMessageWhich(containsString("An error occurred copying file"))));
```